<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileSearcher</class>
 <widget class="QMainWindow" name="fileSearcher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>请输入搜索项</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searcherInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="searcherBtn">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>结果</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="searcherRes"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
